TUT_BY_page.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using OpenQA.Selenium;
+
+namespace AutomationTut.by
+{
+    public class TUT_BY_page
+    {
+        private IWebDriver driver;
+
+        public TUT_BY_page(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        internal protected IWebElement searhField => driver.FindElement(By.XPath("//input[@id='search_from_str']"));
+
+        string login = "aliaksandranovikava@tut.by";
```

Comment:
Try the following:
public string loggin = driver.FindElement(By.XPath("//a[@data-target-popup ='authorize-form']"));